app/models/event_log.rb

```diff
@@ -21,6 +21,7 @@ class EventLog < ActiveRecord::Base
   EMAIL_CHANGE_INITIATED = "Email change initiated"
   EMAIL_CHANGE_CONFIRMED = "Email change confirmed"
   TWO_STEP_ENABLED = "2-step verification enabled"
+  TWO_STEP_DISABLED = "2-step verification disabled"
```

Comment:
I feel like `disabled` should be `reset`, as that's the action that admins are clicking on. Sorry for pedantry :/